docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+ [The global GC cycle is split into 3 increments](./vgclog.md#gc-increments-and-interleaving) that interleave with partial GC cycles. Splitting the cycle operations into the following increments means that the STW pause times can be as short as possible, with the majority of the GC work being done concurrently. The interleaving can be seen in the following example, where a partial GC cycle is logged between the start and end of the global cycle.
+
+ The following elements log the GC increments and operations of the global GC cycle:
+
+ |GC Operation         | GC increment| STW or concurrent| XML element of GC increment| Details                         |
+|---------------------|-------------|-------------------------------|--------------------------------------|-----------------------|
+|n/a - initiates cycle|initial      | STW              | `<concurrent-kickoff`        |No `<gc-op>` is logged. This increment just initiates the concurrent mark increment |
+|concurrent mark      |intermediate |concurrent                     | `<gc-start>`, `<gc-end>`     |`<concurrent-trace-info>` records the progress of the concurrent mark increment|
+|final collection     |final        | STW              | `<concurrent-global-final>`  |Triggered when the card-cleaning threshold is reached. Child element is `<concurrent-trace-info reason=””>`. Operations and sub-operations include a final concurrent mark, a sweep, and an optional class unload and compact.|
+
+The global GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and attributes and some child elements are omitted for clarity. The structure includes a partial GC cycle that starts and finishes within the global cycle:
+
+```xml
+
+<concurrent-kickoff/>       (global cycle 1st increment recorded)
+
+<exclusive-start/>          (STW pause starts)
+
+<cycle-start/>              (global cycle starts)
+
+<exclusive-end/>            (STW pause ends)
+
+
+(global cycle 2nd increment running concurrently)
+
+
+<exclusive-start/>          (STW for partial GC cycle starts)
+
+
+...                         (partial GC cycle starts and completes)
+
+
+<exclusive-end/>            (STW for partial GC cycle ends)
+
+<exclusive-start/>          (STW pause starts)
+
+<concurrent-global-final/>  (global cycle final increment recorded)
+
+<gc-start/>                 (global cycle final increment starts)
+
+<allocation-stats/>
+
+<mem-info>                  (memory status before operations)
+
+<mem></mem>                 (status of different types of memory)
+
+</mem-info>
+
+</gc-start>
+
+<gc-op> “type=rs-scan"</gc-op>        (roots scanning completed)
+
+<gc-op>”type=card-cleaning" </gc-op>  (card cleaning completed)
+
+<gc-op> “type=mark”</gc-op>           (mark operation completed)
```

Comment:
you can be more specific and say: _final_ marking (what really is similar to trancing) and also add weak roots processing